source/extensions/filters/common/fault/fault_config.cc

```diff
@@ -45,6 +50,25 @@ FaultAbortConfig::HeaderAbortProvider::statusCode(const Http::HeaderEntry* heade
   return ret;
 }
 
+absl::optional<Grpc::Status::GrpcStatus>
+FaultAbortConfig::HeaderAbortProvider::grpcStatusCode(const Http::HeaderEntry* header) const {
+  absl::optional<Grpc::Status::GrpcStatus> ret = absl::nullopt;
+  if (header == nullptr) {
+    return ret;
+  }
+
+  uint64_t code;
+  if (!absl::SimpleAtoi(header->value().getStringView(), &code)) {
+    return ret;
+  }
+
+  if (code >= 0 && code <= 16) {
```

Comment:
it's an unsigned integer (uint64_t) - it cannot be lesser than 0, no need to do >= 0 comparison